packages/eslint-config-elixir/package.json

```diff
@@ -13,7 +13,8 @@
     "@typescript-eslint/eslint-plugin": "^5.59.7",
     "@typescript-eslint/parser": "^5.59.7",
     "eslint": "^8.41.0",
-    "eslint-config-airbnb-base": "^15.0.0",
-    "eslint-config-turbo": "1.9.9"
+    "eslint-config-standard": "^17.1.0",
```

Comment:
I think it might be worth considering/enforcing [`eslint-config-airbnb-typescript`](https://www.npmjs.com/package/eslint-config-airbnb-typescript).